doc/howto/gh-actions-badges.md

```diff
@@ -0,0 +1,84 @@
+# Use GitHub Actions To Place A MyBinder.org Badge On Pull Requests
+
+You can facilitate review of GitHub pull requests on [mybinder.org](https://mybinder.org) by using [GitHub Actions](https://github.com/features/actions) to place a ![Binder](https://mybinder.org/badge_logo.svg) badge or link on Pull Requests.  Below are several examples of how to accomplish this.
+
+To enable GitHub Actions, you must place .yaml files that define your GitHub Action workflow in the `/.github/workflows/` directory in your repository.
+
+## Example 1: Comment on pull request with a binder badge
+
+_Note: this example will not work on pull requests from forks.  This is to protect repositories from malicious actors.  To trigger a GitHub Action with sufficient permissions to comment on a pull request from a fork, you must trigger the action via another event, such as a comment or a label.  This is demonstrated in Example 2 below._
+
+The below action uses the [github/script](https://github.com/actions/github-script) Action to call the [GitHub API](https://docs.github.com/en/rest/reference/issues#comments) for the purposes of making a comment on a PR that looks like this:
+
+> ![Binder](https://mybinder.org/badge_logo.svg) 👈 Launch a binder notebook on this branch
+
+```yaml
+#/.github/workflows/binder-badge.yaml
```

Comment:
Does the myst parser let you include the contents of an external file? This would make it slightly nicer for a user- they could download the file instead of copy and paste.

Reply:
@choldgraf do you know the answer to this question, I could not seem to find anything on this

Reply:
Yes! There are two ways to do this in Sphinx that I know of (I am assuming here that the file is local)

If you wish to put the text in an external file, and then insert that text directly into the page, you can use the `include` directive. In MyST markdown, that looks like:

````
```{include} path/tofile.ext
```
````

If you wish to instead add a **download link** to that file, you can use the `download` directive, like so:

````
```{download} path/tofile.ext
```
````

I believe that's how they work, lemme know if that doesn't fix it!

Reply:
wow.  MyST is pretty cool

Reply:
actually Sphinx is pretty cool :-)  MyST is just a markdown syntax to let folks use sphinx features!

Reply:
Sadly, I couldn't make this work 

When I try to use 

```{include} ./binder-badge.yaml
```

It renders the YAML in a weird way 

![image](https://user-images.githubusercontent.com/1483922/87811420-85073f80-c813-11ea-86fc-4380d6f9b86f.png)


After reading the docs a bit more and trying to dig around, I suspect I might need to chain multiple directives, perhaps a directive for a `code` directive?  But I am not sure

 The directive 

```{download} path/tofile.ext
```

Did not render anything unfortunately, only the `include` seemed to work for me

Reply:
ah - sorry I didn't realize you were including a literal block, my bad. I think you can do this with the `literalinclude` directive. Here are the docs in Sphinx (the examples are in rST so you need to translate to MyST, let me know if I can help w/ that): https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude

Reply:
@choldgraf with `literalinclude` I loose syntax highlighting.  Also still cannot find a `download` directive that works, should I keep this the way it is or do you know of what the download one might be?  

![image](https://user-images.githubusercontent.com/1483922/87823939-36fd3680-c829-11ea-9bf5-4559e478e654.png)

I tried to find `download` in the restructured text directives you linked above, and found this https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html?highlight=download#role-download but not sure how I can translate this into MyST.  Do you have any tips on how I should read the docs to try to map these rest directives to things I can use in MyST?  Apologies in advance for my noob questions

Reply:
Edit I figured it out 

```{literalinclude} ./binder-badge.yaml
---
language: yaml
---
```

Still not sure about `download`, though

Reply:
arg my bad - it looks like `download` is  **role**, not a **directive**. Role are in-line versions of directives that have less-flexibility but are more compact. In MyST here's how you write a role:

```
Some markdown text {rolename}`role content` some other markdown text
```

so it would be something like:

```
See this download {download}`path/to/file.yml` for example.
```

I think you can also insert your own text in the link like so:

```
See {download}`this download link <path/to/file.yml>` for example.
```